tests/testthat/test-onevax_xvwr.R

```diff
@@ -26,16 +26,17 @@ test_that("run_onevax_xvwr works correctly", {
     expect_equal(y2[[i]]$T[length(tt), , ], y3[[i]]$T[1, , ])
   }
 
-  uptake <- c(0.5, 1)
+  primary_uptake <- booster_uptake <- 0.5
   # check VoD is working correctly
   y3e <- run_onevax_xvwr(tt, gp, vea = 1, dur = 1e3, strategy = "VoD",
-                         uptake = uptake)
+                         primary_uptake = primary_uptake,
+                         booster_uptake = booster_uptake)
```

Comment:
```suggestion
                         primary_uptake = primary_uptake)
```
check we don't need to set booster uptake and it defaults to be the same as primary